src/app/services/auth-service.service.ts

```diff
@@ -1,42 +1,51 @@
-import { Injectable} from '@angular/core';
-import {Auth, createUserWithEmailAndPassword, signInWithEmailAndPassword, signOut} from "@angular/fire/auth";
+import {inject, Injectable} from '@angular/core';
+import {Auth, createUserWithEmailAndPassword, signInWithEmailAndPassword, signOut, user} from "@angular/fire/auth";
 import { Router } from '@angular/router';
+import {Observable} from "rxjs";
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthServiceService {
 
+  auth: Auth = inject(Auth);
 
+  user$ = user(this.auth);
 
   constructor(
     private router: Router,
     private authentication: Auth
   ) {
-
   }
 
-  // Sign in with email/password
+  get isLoggedIn(): boolean {
+    const user = JSON.parse(localStorage.getItem('user')!);
```

Comment:
The `user` item is not guaranteed on localStorage. Why do you tell typescript it definitely exists?

Reply:
If it doesn't exist just return false